<launch>

  <node pkg="capra_jaus" name="jaus_agent" type="jaus_agent.py" output="screen">
    <param name="judge_server_address" value="192.168.0.110"/> <!-- 192.168.1.42 -->
    <param name="judge_server_port" value="3794"/> <!-- 3794 -->
    <param name="jaus_agent_address" value=""/> <!-- 192.168.1.105 -->
    <param name="jaus_agent_port" value="3794"/>
  </node>

  <node pkg="capra_jaus" name="jaus_client" type="jaus_client.py" output="screen">
    <param name="jaus_node_address" value=""/>
    <param name="jaus_node_port" value="3794"/>
    <param name="multicast_address" value="239.255.0.1"/>
  </node>

  <node pkg="capra_jaus" name="jaus_events_manager" type="jaus_events_manager.py" output="screen">
  </node>


  <node pkg="capra_jaus" name="jaus_access_control_service" type="jaus_access_control_service.py" output="screen">
    <param name="default_authority" value="0"/>
  </node>

</launch>
